### ISSUE TITLE
New Timing part.2

### ISSUE THREAD

**Issue by macanhajc** [macanhajc/my-time#22]
Depends on #17 

## How it works ?
Now is the real shit, we will need to create 3 pages to register a new timing.

### First page
The first page will contains the base info about the timing.

- Title: Title of the timing.
- Description: Description about the timing.
- Schedule: The schedule, from - until (only hours and minutes of the **DAY**)
- Repeat: The repeat is the **DATE** of the timing, could be on specify date like from 10/6 until 10/5 **OR** daily (repeat every day until removed) **OR** today (repeat **ONLY** today) **OR** select the days of the week (repeat wednesday **and** thursday **and** friday) **THIS COMPONENT WILL BE CREATED ON ANOTHER ISSUE**

### Second Page
The second page is just a list of items with an add button and a float button to finish the creation of the timing.

**DO NOT CARE ABOUT THE LAYOUT OF THE BUTTON NOW IT WILL BE IN ANOTHER ISSUE**

- Add button: Will navigate to the **third page**.
- Float button: It concludes the creation of the new timing, it will appears only when the list has one or more itens.

### Third Page

The third page is for create the tasks of the timing,
Example: The timing is Workout, the task of it is to do **x** pushups.

- Title: The title of the task.
- Annotation: Annotation of the task.
- Repeat: How much times you are gonna do that task.
- Time per session: How much time you are gonna spend on that task.

On finish, back to the second page.

![image](https://user-images.githubusercontent.com/19419536/83701282-91c33180-a5df-11ea-98bb-73a9ee699463.png)

### The input
Create the input component.

### First page
- [ ] Create the First page (use PageWrapper).
- [ ] Add the formik and the yup validation for the inputs.
- [ ] Next Button.

### Second page
- [ ] Create the Second page (use PageWrapper).
- [ ] Create list component for the cards.
- [ ] Create the add button wich navigate to the third page.
- [ ] Add the `actionProps` on the pageWrapper, only shows the button when the list has more than one card.

### Third page
- [ ] Create the Third page (use PageWrapper).
- [ ] Add the formik and the yup validation for the inputs.
- [ ] Finish Button, redirects to the second page.
